assets/collage.css

```diff
@@ -249,3 +262,94 @@
     margin-right: var(--card-border-width);
   }
 }
+
+.collage--collapsed .collage__item {
+  --top-left-radius: 0;
+  --top-right-radius: 0;
+  --bottom-right-radius: 0;
+  --bottom-left-radius: 0;
+}
+
+@media screen and (min-width: 750px) {
+  .collage--collapsed .collage__item:only-child,
+  .collage--collapsed-x .collage__item--left:first-child {
+    --top-left-radius: var(--card-corner-radius);
+    --bottom-left-radius: var(--card-corner-radius);
+  }
+
+  .collage--collapsed .collage__item:only-child,
+  .collage--collapsed-x .collage__item--right:last-child {
+    --top-right-radius: var(--card-corner-radius);
+    --bottom-right-radius: var(--card-corner-radius);
+  }
+
+  .collage--collapsed-x .collage__item--right:first-child {
+    --top-left-radius: var(--card-corner-radius);
+  }
+
+  .collage--collapsed-x .collage__item--left:nth-child(2) {
+    --top-right-radius: var(--card-corner-radius);
+  }
+
+  .collage--collapsed-x .collage__item--left:last-child {
+    --bottom-right-radius: var(--card-corner-radius);
+  }
+
+  .collage--collapsed-x .collage__item--right:nth-last-child(2):first-child,
+  .collage--collapsed-x .collage__item--right:nth-child(3n - 1):not(:last-child),
+  .collage--collapsed-x .collage__item--right:nth-child(3n - 2):not(:first-child)  {
+    --bottom-left-radius: var(--card-corner-radius);
+  }
+}
+
```

Comment:
That should be fixed. I also tried to account for mixed use of standard cards and media cards. Like if a standard card sits above a media card with 0px y spacing, the media card should still have radius along the top. I'm a little torn about adding the additional lines of code for it, but since I mostly can solve for it in this section (because of the limited layout options) I figured I should.